src/Selenium2Library/keywords/_formelement.py

```diff
@@ -180,6 +180,26 @@ def choose_file(self, locator, file_path):
                         % file_path)
         self._element_find(locator, True, True).send_keys(file_path)
 
+    def choose_multiple_files(self, locator, basePath="",  *files):
+        """Inputs the `files` into file input field found by `locator`.
+        This Keyword extend the Choose Files option to allow user to select multiple files
+        (input type=files multiple).
+        The file paths need to be specified for each file.
+        Alternative is use basePath argument and then only files name.
+        The file specified with `file_path` must be available on the same host
+        where the Selenium Server is running.
+        Example:
+        | Choose Multiple Files | my_upload_field | /home/user/files/  |  trades.csv,trades.xsl,trades.doc |
+        | Choose Multiple Files | my_upload_field |                    | /folder/trades.csv,/folder1/trades.xsl,/folder3/trades.doc |
+        """
+        fileList = []
+        for file in files:
+            path_to_image = os.path.join(basePath, file)
+            if not os.path.isfile(path_to_image):
+                raise AssertionError("File '%s' does not exist on the local file system" % path_to_image)
+            fileList.append(path_to_image)
+        self._element_find(locator, True, True).send_keys("\n".join(fileList))
```

Comment:
Already added a change for that :) sorry its still WIP but i think i'll submit the new version today.

Also, actually the multiple file load via geckodriver doesn't trigger "onChange()" action.

I've opened a bug so they are working on it :
https://github.com/mozilla/geckodriver/issues/101